markdown/bugs/index.md

```diff
@@ -73,42 +71,62 @@ For troubleshooting the Iris installation, see the [Troubleshooting page][downlo
 
-  * When the user deletes a model component in the model editor window, Iris automatically renames and renumbers the remaining model components. This may oblige the user to then rewrite the model expression. For example, if the user creates the models:
+  * Depending on the size of the Visualizer window, legend might spill over the bounds of the Visualizer when many segments are displayed. 
+    **Workaround:** users can hide the legend from the "View" menu.
+  
+  * When analyzing SEDs with more than 16 segments, fitting ranges are not visualized in the plotter. However, they are still listed in the Fitting Ranges window.
 
-		powerlaw.c1
-		brokenpowerlaw.c2
-		logparabola.c3
-	   
-    and defines the model expression as "c2+c3", then the model that is fit to the data is a broken power-law plus a log-parabola model. If the user deletes the power-law component c1, then the remaining models are automatically renamed:
-   
-		brokenpowerlaw.c1
-		logparabola.c2
-	   
-  * The model expression, however, remains "c2+c3" now a log-parabola model plus a model component that does not exist. The user must rewrite the model expression as "c1+c2" before proceeding with the fit. 
+  * Deleted model components are not removed automatically from the model expression. 
 
   * Highlighting and deleting multiple models from the Components list simultaneously may result in the deletion of unhighlighted models. Delete models from the Components list one-at-a-time for desirable results. 
 
-  * Fitting sessions for multiple SEDs are allowed in Iris. However, switching between sessions (selecting the SEDs currently being fit from the SED Builder) may cause the "Statistics" drop-down menu to disappear. To bring the "Statistics" menu back, close the "Fit Statistics and Optimization Methods" window, and reopen it from the main Fitting Tool window (i.e., click the "Fit" button). 
 
-  * When switching from a populated SED to an empty SED, the Viewer and the Fit windows will not update.
+### <a name="tried-calling-sherpa"></a> "Tried calling Sherpa 100 times every 100 milliseconds. Giving up"
+
+This error may occur when doing one of the following operations:
+  
+  * redshifting
+  * interpolating
+  * calculating integrated fluxes
+  * fitting
+  * calculating confidence intervals
+  * evaluating the model
+  
+If this happens, it is possible that the Sherpa connection stalled out. To remedy this issue, open a new terminal, kill any currently-running `sherpa-samp` processes, start a new `iris` conda environment and launch a new `sherpa-samp` process.
+     
+        $ pkill sherpa-samp
+        $ bash
+        $ source activate iris    # or whatever you've named the environment
+        (iris) $ sherpa-samp
+        
+Then try to do the operation again. You shouldn't lose any of your data if you see this error.
+
+**NOTE:** When you are finished with Iris, you will need to end the sherpa-samp process manually with
```

Comment:
I don't think this is true anymore, since the launcher script should take care of it. However, there is always the possibility that some processes cannot be killed for whatever reason.